packages/shared/templates/content/index.marko

```diff
@@ -86,7 +87,7 @@ $ const displayPublishedDate = ["event", "webinar", "contact"].includes(type) ?
             </default-theme-page-contents>
 
             <aside class="col-lg-4 page-rail">
-              <shared-inquiry-block content=content />
+              <shared-inquiry-block content=content lang=lang/>
```

Comment:
@Shinsina @brandonbk this _should_ have been a lint error -- you need a space between `lang` and the closing of the marko tag.